### PR TITLE
fix(suggest): improve title accessibility

### PR DESCRIPTION
**Pull request by mhagape**

### CODE REVIEW

File: projects/angular/components/ui-suggest/src/ui-suggest.component.spec.ts [hunk 1]
```diff
@@ -1757,7 +1771,8 @@ const sharedSpecifications = (
     });
 };
 
-describe('Component: UiSuggest', () => {
+// tslint:disable-next-line: no-focused-tests
+fdescribe('Component: UiSuggest', () => {
```
Comment (anbalase) [inline projects/angular/components/ui-suggest/src/ui-suggest.component.spec.ts:1775]:
```suggestion
describe('Component: UiSuggest', () => {
```

File: projects/angular/components/ui-suggest/src/ui-suggest.component.html [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
                         !readonly ?
                         0 : null"
          [uiAutofocus]="!isOpen && focus"
          [refocus]="!isOpen && focus"
          [matRippleDisabled]="isFormControl"
+         [attr.aria-label]="placeholder"
+         [attr.aria-describedby]="describedBy"
+         [attr.aria-required]="required"
+         [attr.aria-disabled]="disabled || readonly"
+         [attr.aria-expanded]="isOpen"
```
Comment (anbalase) [inline projects/angular/components/ui-suggest/src/ui-suggest.component.html:15]: 👏

File: projects/angular/components/ui-suggest/src/ui-suggest.component.spec.ts (hunk 1 above)
Comment (mhagape) [reply projects/angular/components/ui-suggest/src/ui-suggest.component.spec.ts]: 🤦  I need a linting rule for this.

**Review by anbalase — APPROVED** [review]
👍

**Pull request merged**